openapi-cli/src/test/resources/generators/client/ballerina/combination_of_apikey_and_http_oauth.bal

```diff
@@ -115,7 +117,9 @@ public isolated client class Client {
         }
         resourcePath = resourcePath + check getPathForQueryParam(queryParam);
         map<string|string[]> httpHeaders = getMapForHeaders(headerValues);
-        Pet response = check self.clientEp->delete(resourcePath, httpHeaders);
+        http:Request request = new;
+        //TODO: Update the request as needed;
+        Pet response = check self.clientEp->delete(resourcePath, request, headers = httpHeaders);
```

Comment:
Shall we avoid creating an empty request here since its an defaultable parameter anyway?